tools/authoring-decisions.adoc

```diff
@@ -307,9 +311,22 @@ The **Decision Service** node needs to be selected in order to see its context m
 It is not enough to hover over the **Decision Service** node, as it is for other DMN node types.
 
 
-.Unkown node
+===== Unknown node
+
+Is a node with an invalid DMN element reference(`dmnElementRef`).
+We cannot render such a node properly as the reference couldn't be found on the current model.
+For such nodes, users will see a placeholder node.
 
-. TODO
+image::tools/authoringDecisions/new-editor-unknown-node.png[]
+
+(1) Reference is the `dmnElementRef`` value that could not resolve to a node.
+The editor does not provide a feature to change this value.
+To change this value, open and edit the file manually outside the editor.
+Other possible fixes are including a model that contains a node with `id` equal to the given `dmnElementRef` value or defining such a node in the current model.
+The `dmnElementRef` has a `QName` value, with the `namespaceDeclarationName:id` syntax.
+With elements on the current model, it's not necessary to add the `namespaceDeclarationName:` hence the `id`.
+On the other hand, models that have included models will have the `namespaceDeclarationName` declared, e.g. `xmlns:included0="<namespace>"`.
+Adding external elements of this included model will result on `dmnElementRef="included0:id"`.
```

Comment:
@jomarko As mentioned on my reply, I don't think this should be added to the docs, at least not this way. Could you please revisit this section?

Reply:
@ljmotta thank you, I pushed new version!